definitions/variable/emissions/tag_ocean-removal-option.yaml

```diff
@@ -0,0 +1,7 @@
+- Ocean Removal Option:
+    - Ocean Alkalinity Enhancement:
+        description: increasing ocean alkalinity
+    - Direct Ocean Capture:
+        description: direct ocean capture
+    - Fertilization:
+        description: ocean fertilization and artificial upwelling
```

Comment:
"Stimulation of the primary production of phytoplankton and other organisms which contribution to ocean uptake of CO2 through direct nutrient addition or artificially upwelling deep nutrient-rich water"